pyproject.toml

```diff
@@ -66,7 +66,7 @@ run_disdrodb_l0b_concat="disdrodb.l0.scripts.run_disdrodb_l0b_concat:run_disdrod
 run_disdrodb_l0b="disdrodb.l0.scripts.run_disdrodb_l0b:run_disdrodb_l0b"
 run_disdrodb_l0="disdrodb.l0.scripts.run_disdrodb_l0:run_disdrodb_l0"
 # Data transfer
-download_disdrodb_archive="disdrodb.data_transfer.scripts.download_disdrodb_archive:download_disdrodb_archive"
+download_disdrodb_archive="disdrodb.data_transfer.scripts.transfer_disdrodb_archive:download_disdrodb_archive"
```

Comment:
Why the name change? 
Note that you can not add the `upload_disdrodb_archive` function in this script !

Reply:
ok